tools/dockerfreeze

```diff
@@ -1,15 +1,22 @@
 #!/bin/sh
+
+# shellcheck shell=bash
+# Affected by https://github.com/koalaman/shellcheck/issues/1771 for directives comments
+
 _dockerfreeze() {
+    # shellcheck disable=SC2034 # FIXME: Are cmds used?
     local cur prev opts cmds
```

Comment:
@JesterOrNot Do you need `cmds` here?

Reply:
no see https://github.com/gitpod-io/dockerfreeze/blob/f61e4d98ec8121629984fd4971c99afad0ef6235/tools/dockerfreeze